.bash_prompt

```diff
@@ -164,7 +164,14 @@ sexy_bash_prompt_branch_exists () {
 sexy_bash_prompt_parse_git_ahead () {
   # Grab the local and remote branch
   branch="$(sexy_bash_prompt_get_git_branch)"
-  remote_branch="origin/$branch"
+  
+  # remote_branch="origin/$branch"
+  # Compute the remote tracking branch, if no remote branch then echo 0
+  remote_branch="$(git rev-parse --symbolic-full-name "@{upstream}" 2>/dev/null || echo -n "origin/$branch")"
+  if [[  "$remote_branch" == "" ]]
+    then
+      echo 0; return;
+  fi
```

Comment:
There is never no remote, it's either set via upstream tracking or we are falling back to `origin`. When the `origin` remote isn't set, we always state that code is unpushed (and the code aligns with that by the `log` failing)